src/aether-elasticsearch-consumer/values.yaml

```diff
@@ -22,3 +27,8 @@ app:
   elasticsearch:
     url: "http://127.0.0.1:9200"
     user: admin
+
+  redis:
+    db: 1
```

Comment:
I don't know how you want to manage database #s for Redis.

Reply:
did you want to change this to `0` to keep it default